include/internal/thread_arch.h

```diff
@@ -12,25 +12,23 @@
 # include <openssl/configuration.h>
 # include <openssl/e_os2.h>
 # include <internal/time.h>
-
```

Comment:
Spurious?

Reply:
enh.  i glued together all the include's, even if one was ifdef'd.